dts/bindings/spi/spi.yaml

```diff
@@ -19,6 +19,11 @@ properties:
       category: required
       description: compatible strings
       generation: define
+    clock-frequency:
+      type: int
+      category: optional
+      description: Clock frequency information for UART operation
```

Comment:
Whoops, missed the description line when I copied this from the generic UART binding.